code/game/jobs/job/engineering.dm

```diff
@@ -32,11 +32,8 @@
 		H.equip_or_collect(new /obj/item/clothing/head/hardhat/white(H), slot_head)
 		H.equip_or_collect(new /obj/item/weapon/storage/belt/utility/full(H), slot_belt)
 		H.equip_or_collect(new /obj/item/clothing/gloves/color/black/ce(H), slot_gloves)
-		if(H.backbag == 1)
-			H.equip_or_collect(new /obj/item/weapon/storage/box/engineer(H), slot_r_hand)
-		else
-			H.equip_or_collect(new /obj/item/weapon/storage/box/engineer(H.back), slot_in_backpack)
-			H.equip_or_collect(new /obj/item/weapon/melee/classic_baton/telescopic(H.back), slot_in_backpack)
+		give_kit(H)
+		H.equip_or_collect(new /obj/item/weapon/melee/classic_baton/telescopic(H.back), slot_in_backpack)

```

Comment:
Actually, I think I could get away with just switching to this method for every single item that's put into a backpack. Much cleaner.